haystack/components/extractors/named_entity_extractor.py

```diff
@@ -259,16 +257,14 @@ def __init__(
     @abstractmethod
     def initialize(self):
         """
-        Initializes the backend. This would usually
-        entail loading models, pipelines, etc.
+        Initializes the backend. This would usually entail loading models, pipelines, etc.
```

Comment:
```suggestion
        Initializes the backend. This would usually entail loading models, pipelines, and so on.
```

Avoiding latinisms :)